addons/sale_timesheet/static/tests/mock_server/mock_models/project_task.js

```diff
@@ -0,0 +1,5 @@
+import { models } from "@web/../tests/web_test_helpers";
+
+export class ProjectTask extends models.ServerModel {
+    _name = "project.task";
```

Comment:
models should be defined in their main module otherwise we might run into some strange dependency issues if current module is not installed